dropwizard-demo/test/com/test/ControllerTest.java

```diff
@@ -103,28 +79,6 @@ public void testcase_sendTweet_success() {
         Assert.assertTrue(T);
     }
 
-    @Test
-    public void testcase_searchTweets() throws TwitterException {
-        when(brsConfiguration.configurationBuilder()).thenReturn(new ConfigurationBuilder());
-        ArrayList<String> tweet = new ArrayList<String>();
-        tweet.add("hlo");
-        tweet.add("hi");
```

Comment:
here too , whats happening

Reply:
This testcases are related to duplicate code of my GET method. This testcases is getting repeated , so I have deleted this testcases.